src/pages/organizers/create/OrganizerForm.tsx

```diff
@@ -72,6 +99,20 @@ const OrganizerForm = () => {
           form={form}
         />
       </Page.Content>
+      <Page.Footer>
+        <Button
+          variant={ButtonVariants.SUCCESS}
+          onClick={async () =>
```

Comment:
I would use react-form-hook's `handleSubmit` here so it triggers validation etc?

Reply:
Yeah this is just a draft PR so I stopped there but definitely not final iteration 😅